bitnami/harbor/values.yaml

```
@@ -3701,6 +3716,251 @@ trivy:
       http: 8080
       https: 8443
 
+## @section Harbor Exporter Parameters
+
+exporter:
+  ## Bitnami Harbor Exporter image
+  ## ref: https://hub.docker.com/r/bitnami/harbor-exporter/tags/
+  ## @param exporter.image.registry Registry for exporter image
+  ## @param exporter.image.repository Repository for exporter image
+  ## @param exporter.image.tag Tag for exporter image
+  ## @param exporter.image.pullPolicy Harbor exporter image pull policy
+  ## @param exporter.image.pullSecrets Specify docker-registry secret names as an array
+  ## @param exporter.image.debug Specify if debug logs should be enabled
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/harbor-exporter
+    tag: 2.4.1-debian-10-r0
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+    ## Set to true if you would like to see extra information on logs
+    ## It turns BASH and/or NAMI debugging in the image
+    ##
+    debug: false
```

Nami doesn't exist anymore:
```suggestion
    ## Enable debug mode
    ##
    debug: false
```